dev_guide/compute_resources.adoc

```diff
@@ -404,6 +417,23 @@ spec:
         cpu: 2
         pod.alpha.kubernetes.io/opaque-int-resource-foo: 1
 ----
++
+The pod will be scheduled only if all of the resource requests are satisfied
+(including CPU, memory, and any opaque resources). The pod will remain in the
+`PENDING` state while the resource request cannot be met by any node
++
+[source,bash]
+----
+Conditions:
+  Type    Status
+  PodScheduled  False
+...
+Events:
+  ...
+  ... Warning   FailedScheduling  pod (oir-demo-2) failed to fit in any node
+fit failure summary on nodes : Insufficient pod.alpha.kubernetes.io/opaque-int-resource-dongle (1)
```

Comment:
It'd better if they have the pod spec mentioned above. As I dont oir-demo-2 and opaque-int-resource-dongle mentioned anywhere above.